apps/nextra/pages/en/build/create-aptos-dapp.mdx

```diff
@@ -31,7 +31,7 @@ cd your/workspace
 
 ### Install `create-aptos-dapp`.
 
-<Tabs items={['npx', 'pnpx']}>
+<Tabs items={['npx', 'pnpx', 'yarn', 'pnpm']}>
```

Comment:
> Pnpx is already covered in the items array

In most cases, we should allow users to choose from a variety of methods. Not everyone is familiar with npx or pnpx, so I believe we should offer more options for them to choose from.

Reply:
@WGB5445 `pnpm create-aptos-dapp@latest` does not work because it expects an existing `package.json` with that script available. Gives you an error like so:

```bash
 ERR_PNPM_NO_IMPORTER_MANIFEST_FOUND  No package.json (or package.yaml, or package.json5) was found in "/Users/<USER>".
```

You need to do `pnpx` or `npx`.